src/output/png.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+namespace GOL\Output;
+
+require_once "seasonal.php";
+
+use GOL\Boards\Board;
+use Ulrichsg\Getopt;
+
+/**
+ * Saves the Board as a png sequence.
+ *
+ * Use write() to write the Board data to the output.
+ * and flush() to write it on the disc.
+ */
+class PNG extends Output
+{
+    private $buffer = [];
+    private $cellSize = 1;
+    private $backgroundColor = [];
+    private $cellColor = [];
+
+    /**
+     * Writes the current board to the Output.
+     *
+     * @param Board $_board Board to output.
+     */
+    public function write(Board $_board): void
+    {
+        $this->buffer[] = $_board->getGrid();

```

Comment:
Here it would be even better to create an image on every `write()`.
If you calculate 10000 generations and then your memory is exhausted you have zero output, whereas if you output on every write() you at least would have the output of all frames that where calculated already.